Makefile

```
@@ -337,7 +337,7 @@ $(DIST_DIR)/%/vmclarity-cli: $(shell find api cli utils core)
 	GOOS=$(firstword $(subst -, ,$*)) \
 	GOARCH=$(lastword $(subst -, ,$*)) \
 	CGO_ENABLED=0 \
-	go build -ldflags="$(LDFLAGS)" -o $@ cmd/$(notdir $@)/main.go
+	go build -C cli/cmd -ldflags="$(LDFLAGS)" -o $@ main.go
```

```suggestion
	go build -C cli -ldflags="$(LDFLAGS)" -o $@ cmd/main.go
```
Go build is better to be run from the module directory (where the `go.mod` file is).